tools/mason/Makefile

```diff
@@ -36,9 +43,9 @@ mason: $(bdir)/mason FORCE
 
 install: $(bdir)/mason FORCE
 
-$(bdir)/mason: $(MASON_SOURCES)
+$(bdir)/mason: $(MASON_SOURCES) buildMason
 	@echo "Building Mason..."
-	@chplBinDir=$(bdir) ./buildMason
+	@chplBinDir=$(bdir) DEBUG=$(DEBUG) ./buildMason
```

Comment:
(Nothing about your PR). I have a growing wish to be able to build Mason with Mason.